src/main/java/store/service/AlbumServiceImpl.java

```diff
@@ -45,17 +46,18 @@ public void save(Album album) {
     }
 
     @Override
-    public boolean update(Album album) {
-        Optional<Album> albumOptionalFromDB = albumRepository.findById(album.getId());
+    public Album update(Long id, Album album) throws NotFoundException {
+        Optional<Album> albumOptionalFromDB = albumRepository.findById(id);
         if (!albumOptionalFromDB.isPresent()) {
             System.out.println("Album not found");
```

Comment:
Should use a logger here and everywhere. Using logging and throwing at the same time is a bad practice. https://dev64.wordpress.com/2012/10/07/exception-handling-antipatterns/